charts/rancher-eks-operator/1.0.801/templates/deployment.yaml

```diff
@@ -25,3 +25,17 @@ spec:
           value: {{ .Values.httpsProxy }}
         - name: NO_PROXY
           value: {{ .Values.noProxy }}
+        {{- if .Values.additionalTrustedCAs }}
+        volumeMounts:
+          - mountPath: /etc/ssl/certs/ca-additional.pem
+            name: tls-ca-additional-volume
+            subPath: ca-additional.pem
+            readOnly: true
+        {{- end }}
+      {{- if .Values.additionalTrustedCAs }}
+      volumes:
+      - name: tls-ca-additional-volume
+        secret:
+          defaultMode: 0400
+          secretName: tls-ca-additional
```

Comment:
Should this value be providable by the user? Also, should we add this to the documentation (README.md?)

Reply:
That was my original thought, but this follows what is done in Rancher: https://github.com/rancher/rancher/blob/master/chart/templates/deployment.yaml#L136-L145